demisto_sdk/commands/common/MDXServer.py

```diff
@@ -0,0 +1,86 @@
+from abc import abstractmethod
+
+import subprocess
+
+from typing import Optional
+
+import logging
+import os
+from pathlib import Path
+
+import docker
+import docker.errors
+import docker.models.containers
+
+from demisto_sdk.commands.lint.docker_helper import (Docker,
+                                                     init_global_docker_client)
+
+DEMISTO_DEPS_DOCKER_NAME = "demisto-dependencies"
+_SERVER_SCRIPT_NAME = 'mdx-parse-server.js'
+_MDX_SERVER_PROCESS: Optional[subprocess.Popen] = None
+
+
+def server_script_path():
+    return Path(__file__).parent.parent / 'common' / _SERVER_SCRIPT_NAME
+
+
+class DockerMDXServer:
+    def __enter__(self):
+        image_name = 'devdemisto/demisto-sdk-dependencies:1.0.0.33871'
```

Comment:
1. Hard-coded? 
2. Do we ever update it?

Reply:
moved to constants. We will probably update with code changes